robottelo/decorators.py

```diff
@@ -273,12 +274,18 @@ def bz_bug_is_open(bug_id):
     :rtype: bool
 
     """
+    upstream_mode = str(conf.properties.get('main.upstream', 1)) == '1'
```

Comment:
I think that it is not necessary to cast the value returned from `conf.properties.get('main.upstream', 1)` to a string, since the objective is to use it as a boolean flag a bit further down. You could drop this line altogether and perform your check inside the conditional statement below. If `conf.properties.get('main.upstream', 1)` doesn't exist/cannot be found, you end up with the default value `1` which can also be interpreted as `True`. If, however, the value is **not** `1` and as long as it is an integet, it should be interpreted as `False`.